### PR TITLE
push data to DB

### PR DESCRIPTION
**Pull request by BarBenNaim**
A function that inserts data into the database from an API.
The function gets the company name and the 2 lists: a list of titles and a list of links

### CODE REVIEW

File: ultimatejobweb/models.py [hunk 1]
```diff
@@ -13,5 +13,13 @@ class Job(models.Model):
     job_title = models.CharField(max_length=128)
     description_url = models.URLField()
 
+    def push_data_api(company_name_api, title_list, url_list):
```
Comment (simzacks) [inline ultimatejobweb/models.py]: Should this function be part of the Job class?
If so, it should be a class method and have the cls parameter first.
But it doesn't look to me like it belongs here

Comment (Yarboa) [reply ultimatejobweb/models.py:16]: @simzacks where do you think it should be ? 
It is a method to update new jobs

Comment (Yarboa) [reply ultimatejobweb/models.py]: > Should this function be part of the Job class?
> If so, it should be a class method and have the cls parameter first.
> But it doesn't look to me like it belongs here
@simzacks where do you think it should be ?
It is a method to update new jobs

Comment (Yarboa) [reply ultimatejobweb/models.py]: @BarBenNaim  please refer this sample PR
https://github.com/beyond-io/beyond-tutorial/pull/7/files#diff-d674dd14acda98e0ff388c113622f5224b22cde68f320bf7e6d39ce166214c09R11

Comment (BarBenNaim) [reply ultimatejobweb/models.py]: @Yarboa @simzacks 
I don't understand why I need class?

File: ultimatejobweb/models.py [hunk 2]
```diff
@@ -13,5 +13,14 @@ class Job(models.Model):
...
     description_url = models.URLField()
 
+    @classmethod
+    def push_data_api(cls, company_name_api, title_list, url_list):
+        combined = zip(title_list, url_list)
+        combined_list = list(combined)
+        title_list, url_list = zip(*combined_list)
+
+        for job_title, description_url in combined_list:
+            company_job = Company.objects.get(company_name=company_name_api)
```
Comment (Yarboa) [inline ultimatejobweb/models.py:23]: @BarBenNaim 
I think this method should be part of class company
And than you could use the cls member

  for job_title, description_url in combined_list:
            company_job = cls.objects.get(company_name=company_name_api)

@BarBenNaim do you see it now?

Comment (BarBenNaim) [reply ultimatejobweb/models.py:23]: @Yarboa Yes Im try

Comment (BarBenNaim) [reply ultimatejobweb/models.py:23]: `    @classmethod
    def push_data_api(cls, company_name_api, title_list, url_list):
        combined = zip(title_list, url_list)
        combined_list = list(combined)
        title_list, url_list = zip(*combined_list)

        for job_title, description_url in combined_list:
            company_job = Company.objects.get(company_name=company_name_api)
            cls(company=company_job, job_title=job_title, description_url=description_url).save()`

Comment (Yarboa) [reply ultimatejobweb/models.py:23]: Ack

**Pull request merged**